scripts/messages.js

```diff
@@ -30,7 +30,7 @@ let sendMessage = (message, dest, callback) => {
     formatMessage(message, function (res) {
         gdocs.setAuth(dest, function (err, info) {
             if(!err) {
-                gdocs.addRow(info.worksheets[0].id, res, function (err, info) {
+                gdocs.addRow(info[1].worksheets[0].id, res, function (err, info) {
```

Comment:
Not really getting this one `info[1]` - is this always the case? what if there is not element 1? will not this `error`?